components/CreateForm/ImageEditor.tsx

```diff
@@ -308,6 +305,30 @@ export default function ImageEditor(): ReactElement {
     });
   }, []);
 
+  const [position, setPosition] = useState({ x: 0, y: 0 });
+  const dragging = useRef(false);
+  const startPos = useRef({ x: 0, y: 0 });
+
+  const handleImageMouseDown = (e: React.MouseEvent) => {
+    dragging.current = true;
+    startPos.current = {
+      x: e.clientX - position.x,
+      y: e.clientY - position.y,
+    };
+  };
```

Comment:
open close principle
- actual: expanding existing component with new code
- required: create new `hooks` / `components` which are imported into the existing code